package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-template-node",
-  "version": "0.5.2",
+  "version": "0.6.0",
```

Comment:
I would prefer to still keep it on the 0.5.x branch, and switch to 0.6.0 once we start the switch to Node 8 in production.